sherpa/astro/data.py

```diff
@@ -107,7 +107,7 @@ def __str__(self):
         old = self._fields
```

Comment:
Given that this is repeated for all the three classes (or at least it seems to be from a quick look), can we not move this it into a separate routine? It's probably not worth the hassles of doing this in a nice OO way, but I thought I'd ask.